services/graphql/src/graphql/resolvers/report.js

```diff
@@ -21,7 +21,6 @@ module.exports = {
     publisher: ({ publisherId }) => publisher.findById(publisherId),
     deployment: ({ deploymentId }) => deployment.findById(deploymentId),
     adunit: ({ adunitId }) => adunit.findById(adunitId),
-    ctr: ({ impressions, clicks }) => clicks / impressions,
```

Comment:
Why remove this? Graph can resolve the ctr... you'll just need to ensure impressions is set and greater than 0

Reply:
It's being resolved in the aggregation query, no need to run the same op twice